content/Hardware Support/Portenta Family/FAQ-Arduino-Opta.md

```diff
@@ -51,6 +60,10 @@ The frequently asked questions covered in this article are focused on providing
 
     PLC IDE offers two cyclical tasks. The Fast task's lowest cycle time, set as default, is 10ms, but it can also be configured to use a 20ms or 50ms execution period. A Slow task is also available, for less priority operations, with a 100ms execution period.
 
+- **Why is the Simulation Mode disabled in PLC IDE?**
+
+    The Simulation Mode is not available in the PLC IDE due to the runtime implementation as, including the Arduino code, it would not be capable of simulating reliably that part.
```

Comment:
```suggestion
    The Simulation Mode is unavailable in the PLC IDE because the runtime implementation, including the Arduino code, would not reliably support simulation for that part.
```